src/Microsoft.Restier.WebApi/Model/ApiModelBuilder.cs

```diff
@@ -11,16 +11,17 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.OData.Edm;
 using Microsoft.OData.Edm.Library;
+using Microsoft.Restier.Core;
 using Microsoft.Restier.Core.Model;
 using Microsoft.Restier.Core.Query;
 
-namespace Microsoft.Restier.Core.Conventions
+namespace Microsoft.Restier.WebApi.Model
 {
     /// <summary>
     /// A convention-based API model builder that extends a model, maps between
     /// the model space and the object space, and expands a query expression.
     /// </summary>
-    internal class ConventionBasedApiModelBuilder
+    internal class ApiModelBuilder
```

Comment:
I can not say for sure now, but if some protocol reuse it, it will make two protocol very similar, like one possible is OneApi, we use same code based to support it and OData.